README.md

```diff
@@ -109,30 +110,30 @@ To clone a project, use the command:
 git clone --recursive https://github.com/Gedsh/InviZible
 ```
 
-To build **InviZible Pro** please use **Android Studio**.
+To build **InviZible Pro**, please use **Android Studio**.
 
 If you see something like this:
 _Illegal character in opaque part at index 2: C:\KStore\keystore.properties_
 
-Please comment lines of the settings.gradle file in the project root, as shown below:
+Please comment out this line in the settings.gradle file of the project root, as shown below:
 
 ```bash
 include ':tordnscrypt', ':filepicker'
 project(':filepicker').projectDir = new File('android-filepicker/filepicker')
-//Please comment line below if you are not the project owner
+//Please comment out the line below if you are not the project owner
 //project(':tordnscrypt').buildFileName = 'owner.gradle'
 ```
 
-It is expected that you can solve build problems yourself, so there is no support for building. 
-If you cannot build yourself, there are prebuilt versions of **InviZible Pro** available [here](https://github.com/Gedsh/InviZible/releases/latest).
+As is expected, if you can solve how to build problems for yourself, so there is no support for how you got there. :) 
+If you cannot build the app yourself, there are pre-built versions of **InviZible Pro** available [here](https://github.com/Gedsh/InviZible/releases/latest).
 
 #### Translating
 
-If you would like to translate this application into other languages,
-please send an email to the developer [invizible.soft@gmail.com](mailto:invizible.soft@gmail.com)
+[Translate InviZible on Hosted Weblate](https://hosted.weblate.org/engage/invizible/).
+
+[![Translation status](https://hosted.weblate.org/widgets/invizible/-/multi-auto.svg)](https://hosted.weblate.org/engage/invizible/?utm_source=widget)
 
-Thanks to [POEditor](https://poeditor.com/), you can translate InviZible Pro into your language with pleasure and for free.
-Just follow this [link](https://poeditor.com/join/project/h6ulNL9gEd).
```

Comment:
Why have you removed POEditor? InviZible continues to support translation with it.

Reply:
POEditor is not a good tool, and advertising it isn't something I want to do.

Reply:
POEditor is a simple tool for anyone to translate. Webplate is very complex.

Reply:
And they all do, except there isn't any completion of languages, and there are very few translators.
There is no community there. People use InviZible first and then use whatever tool it mandates second.

POEditor is a simplistic tool that doesn't provide enough utility to fix the errors it produces.
https://hosted.weblate.org/checks/?project=invizible&component=invizible

Weblate has users, a libre license, it is the better tool, and it produces better results.
As a service, Hosted Weblate has better terms
>CODE WHALE reserves the right to change the Terms from time to time in its sole discretion without any notifications regarding this matter. 

There are no libre software projects using POEditor that I know of. Why should InviZible use or advertise it?

Reply:
> Weblate has users, a libre license, it is the better tool, and it produces better results.

InviZible has been translated into 10 languages without any problem using POEditor. And I have a problem fixing only English translation with Webplate. The result is obvious.

> Why should InviZible use or advertise it?

Because it produces result and a free plan for open source projects. So why not.

Reply:
The 10 languages are full of errors, some of which are incomplete.
There is a nb_NO translation made on Weblate, and it isn't good.
First the source has to make sense, then the translations can make sense.
Sending people to the place that cares for user freedom and happens to be the better software is less complicated and better for translation-quality.

Reply:
I give people the choice to use POEditor or Webplate. For me, choice is freedom. You decided instead of everyone. Is this such modern freedom?

Reply:
The freedom of choice isn't removed. I decide nothing for anyone. That is your all your decision.

Applying the same level of consistency to the app would be to argue for a setting to decrypt all traffic too, albeit at a logical extreme.

Similarly, a website that lists your app next to some shady VPN service, or shoes for sale, isn't promoting more freedom, but less, and with more noise.
To say entropy is tantamount to freedom amounts to practical sophistry, and the amount of available and good choices dwindle from making poor judgements.
Nobody actually wants to use POEditor, yet. No libre software projects actually use it that I know of.
It is neither simple, its results are not good, and the feature is not in demand.

I try not to mislead libre software translators into a only-as-a-service platform that can change their terms whenever they want. It also loads Google Analytics and a bunch of other stuff.
Not everyone knows about that, so using the good name of InviZible to promote a service like that just doesn't serve the project. It may seem like a comparable product when it isn't.

POEditor is the opposite of freedom, and there is not less of it in any meaningful sense if the option is not available, or not advertised.

TL;DR All mentioning POEditor amounts to is to undermine the principles of the app and the better translation platform.

Reply:
I get it. You are an adherent of modern "freedom". It sounds like this:
You are human. Of course, you have a choice. You have several options, and our best experts will select the best one for you. So you actually have only one option. But it is from experts! It is the best! So make your choice from one option.
Thanks so much. I prefer "classic" freedom with real multiple options.

Reply:
<details>
I find it hard to offer a coherent set of options without making the concession that, and i find you agree, some options are less good for good reason. This is the paradox of choice.

Every single libre software project has understood using POEditor is not good PR, bar none? 
The entire ecosystem is not less free because of it.
A further nobody recommends POEditor. Except for this project.
In this project, it appears nobody actually uses POEditor to translate InviZible, but there have been over 400 translations on the non-advertised Weblate platform.

Weblate respects user freedom, on a meaningful level, and it dominates for libre software projects,
without limiting anyone from using other alternatives.

Relying on POEditor in any way means being subject to terms they can change, at any time, and a codebase they can shut down tomorrow. For any appreciation of what freedom is outside of noise, that is important.

Similarly, the GNU GPLv3+ license is not allowing the maximum set of options, it clearly limits bad options to what is a end product that favours user freedom over "I am the user that doesn't want to offer a derivative of the project with the source code"-user.

Having superfluous, excessive, and bad options available contributes to the cost of opportunity.
It creates stress. People have to decide what to do.
I decide I fix my suggestions if there is a good argument, and I assume all other options may be good, but beyond my limit of understanding.

There was one opportunity to use one platform, and now there will be two platforms for no reason, but one will be advertised. Better platform, and more choice.
</details>

Do you think anyone will use POEditor if both are advertised?
Will anyone complain?
Is there any reason to list POEditor rather than the theoretical argument?
In this line of thinking the project should be available and advertised on every platform there is.

Reply:
To add to this argument, why not offer the option to route DNS onto Google's server in the app? Terrible freedom, that doesn't add anything. It is just a worse option, like Poeditor is.
Why are users worth less once they are translators contributing to the app?
Why split the community of translators between different ecosystems?
Why should prospect translators have to decide between two platforms and have to find out which one is the best?

Paradox of overloading users with bad choices does not grant more user freedom.